pipe_js.go

```diff
@@ -20,88 +20,7 @@ import (
 	"io"
 )
 
-const pipeBufSize = 4096
-
-// pipe returns a set of an io.ReadCloser and an io.WriteCloser.
-//
-// This is basically same as io.Pipe, but is implemented in more effient way under the assumption that
-// this works on a single thread environment so that locks are not required.
```

Comment:
Please leave these comments.

Reply:
I think its outdated now, I'll instead update it to reflect the new state of things.